### ISSUE TITLE
App doesn't build for Android and iOS

### ISSUE THREAD

**Issue by alexdoering** [blefebvre/aem-phonegap-starter-kit#42]
Hi, 
I can't get this aem-phonegap-starter-kit (vanilla version) to build properly:
Installing a build which was triggered remotely from AEM to a Phonegap Build account, comes up with an  error message relating to the Cordova Plugins on an Android and iOS device: Device is not defined. I've added a screenshot. 
Building it locally via Phonegap CLI works for iOS but not for Android (due to multiple errors in the generated AndroidManifest.xml).
I've tried some other apps (e.g kitchen sink) but none of them work when a build from Phonegap Build is installed. 
The only app building and installing properly is the Geometrixx Outdoors App. 

I have no idea what the issue is and would be glad for any kind of help on this. Because we have a cool Angular/Ionic App in AEM now and can't get it onto a device :)

Thanks, 
Alex
![screenshot_2015-08-24-13-58-52](https://cloud.githubusercontent.com/assets/386423/9510671/f1995594-4cb1-11e5-9700-655b325651a0.png)

**Comment by blefebvre** [comment]
Hey Alex, 

Sorry to hear that you've run into issues. I've been working away at this repo to get it back on track with the latest CLI and plugins from the PhoneGap team.

I just pushed up a commit that removes all the plugin source code from the repo, and replaces it with the new `<gap:plugin` format that uses plugins from npm. I was able to kick off a remote build and install it on iOS via the provided QR code, and it worked as expected. Note that 'Build Remote' from the PhoneGap Build tile uses the last 'staged' update, so once you install the latest starter-kit source you will need to either clear out both content sync caches ('app_name_placeholder App' and 'English'), or Stage a new update for each.

The plugin source code was originally checked in to facilitate faster builds and make demos possible in offline environments, but I've been advised by the PG team that this is not the recommended model. Kitchen-sink will be updated with the same fixes ASAP (we're seeing weird build issues over there, too: https://github.com/blefebvre/aem-phonegap-kitchen-sink/issues/36). 

Regarding your issue building for Android from the CLI: which version of the CLI are you on? I was able to build and install the starter-kit on Android earlier today with PhoneGap 5.1.1. I'd also be curious if you're able to install a basic PhoneGap helloworld app (created with `phonegap create myapp`).

I'll keep this issue open until I've heard that it's working for you.

**Comment by alexdoering** [comment]
Hi Bruce,

Thanks for your super quick response!
I will check out the updated code and send through my PG CLI details as
soon as I have time (just got shipped off to another project :(

I also saw that the master now has the /feature/ionic-1.0 merged. Anthony
created that feature because I was bugging him so much when I had trouble
integrating Ionic (JS). Anthony's feature branch still didn't work for me
but I had to additionally add 'ionic' to the 'angular-module-list.js.jsp'
so that angular knows about that dependency - know everything works in AEM
author :) Unless it works for you, it might be worth adding this.

Thanks,
Alex
